code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -24,14 +24,12 @@
 /obj/structure/stool/bed/Move(atom/newloc, direct) //Some bed children move
 	. = ..()
 	if(buckled_mob)
-		buckled_mob.buckled = null
 		if(!buckled_mob.Move(loc, direct))
 			loc = buckled_mob.loc //we gotta go back
 			last_move = buckled_mob.last_move
 			inertia_dir = last_move
 			buckled_mob.inertia_dir = last_move
 			. = 0
-		buckled_mob.buckled = src

```

Comment:
This little trick is no longer needed, living/Move() no longer has a specific simple `if(buckled) return` check preventing you from ever moving while buckled. Removing this enables calling M.buckled when `buckled_mob.Move(loc, direct)` makes you cross stuff like wet floor or banana peels, so we can vary the slipping depending on whether we're buckled.